barretenberg/cpp/src/barretenberg/benchmark/client_ivc_bench/client_ivc.bench.cpp

```diff
@@ -46,24 +46,59 @@ class ClientIVCBench : public benchmark::Fixture {
         GoblinMockCircuits::construct_mock_function_circuit(function_circuit);
         ivc.initialize(function_circuit);
 
-        // Accumulate kernel circuit (first kernel mocked as simple circuit since no folding proofs yet)
```

Comment:
You've done the multithreading in a way such that only 5 out of the 6 iterations are being parallelized. Why not start the parallelization with the very first iteration?